portal-impl/src/com/liferay/portal/events/StartupAction.java

```diff
@@ -70,8 +74,20 @@ protected void doRun(String[] ids) throws Exception {
 		// Check Tomcat's lib/ext directory
 
 		if (ServerDetector.isTomcat()) {
-			File libExtDir = new File(
-				PropsValues.LIFERAY_LIB_GLOBAL_SHARED_DIR, "ext");
+			String path = ClassUtil.getParentPath(
```

Comment:
@julschong Please try not to use ClassUtil, let's try to simplify the logic here as much as possible.

Thanks.
Tina.